examples/ffi/c++/main.cpp

```diff
@@ -17,7 +17,7 @@ int main(int argc, char *argv[]) {
         std::cerr << filecoin_signer_error_message(error);
     }
     else {
-        char *private_key = filecoin_signer_extended_key_private_key(extended_key);
+        char *private_key = filecoin_signer_extended_key_private_key(extended_key, error);
```

Comment:
I think we need to discuss this in Slack, it feels a bit strange from a C/C++ perspective.